internal/users/cache/db_test.go

```diff
@@ -62,7 +64,7 @@ func TestNew(t *testing.T) {
 				require.NoError(t, err, "Setup: could not change mode of database file")
```

Comment:
Do not forget: probably new test cases as we don’t clear the database anymore.

Reply:
I haven’t seen new uses cases. I think one is "trying to open a corrupted database" (file is readable, just not the correct format) and assert we return an error.

Ensure then that all etters that return an error on corrupted database have a test checking with it.